test/test_meta.py

```diff
@@ -726,6 +727,7 @@ def run_meta_crossref(
 meta_function_device_skips['cpu'] = {
     torch.native_batch_norm: {f32, f64},
     torch._native_batch_norm_legit: {f32, f64},
+    torch.ops.aten._batch_norm_with_update: {f32, f64},
```

Comment:
Why does the meta function break?

Reply:
This is the error I got:
```
RuntimeError: output 1: meta disagrees with real impl:
aten._batch_norm_with_update.default(...)
for element 1, was torch.float32 but real dtype was torch.bfloat16
```
Basically `running_mean` and `running_var` are not the right dtype, but the rest of the args are. Haven't investigated into the root cause but seems like `_native_batch_norm_legit` also has the same problem